test/integration/targets/vmware_guest_network/tasks/main.yml

```diff
@@ -163,6 +163,36 @@
         - del_again_netadapter is changed
         - "{{ del_again_netadapter.network_data | length | int }} == {{ netadapter_num | int + 1 }}"
 
+  - name: disable DirectPath I/O on a Vmxnet3 adapter
+    vmware_guest_network:
+      validate_certs: False
+      hostname: "{{ vcenter_hostname }}"
+      username: "{{ vcenter_username }}"
+      password: "{{ vcenter_password }}"
+      name: "{{ virtual_machines[0].name }}"
```

Comment:
@goneri 
I had the same concern as well, since it was not related to this patch, I thought of not modifying it. 
Anyway, changing it now. Looks like it needs to be changed in few of the other tasks as well, will do it.